superdesk/places/places_autocomplete.py

```diff
@@ -50,3 +50,21 @@ def get(self, req, lookup):
         json_data = geonames_request("search", params)
         data = [format_geoname_item(item) for item in json_data.get("geonames", [])]
         return ListCursor(data)
+
+    def get_detail(self, req, lookup):
+        assert req.args.get("geonameId"), {"geonameId": 1}
+        params = [
+            ("geonameId", req.args.get("geonameId")),
+            ("lang", req.args.get("lang", "en").split("-")[0]),
+            ("style", req.args.get("style", app.config["GEONAMES_SEARCH_STYLE"])),
+        ]
+
+        if req.args.get("featureClass"):
+            params.append(("featureClass", req.args.get("featureClass")))
+        else:
+            for feature_class in app.config["GEONAMES_FEATURE_CLASSES"]:
+                params.append(("featureClass", feature_class.upper()))
```

Comment:
do you need this? imo that's only for search

Reply:
Good Point.